app/javascript/packs/component-definitions-common.js

```diff
@@ -70,6 +76,12 @@ ManageIQ.component.addReact('TagGroup', props => <TagGroup {...props} />);
 ManageIQ.component.addReact('TagView', TagView);
 ManageIQ.component.addReact('TaggingWrapperConnected', TaggingWrapperConnected);
 ManageIQ.component.addReact('TextualSummaryWrapper', TextualSummaryWrapper);
+ManageIQ.component.addReact('RightSection', RightSection);
```

Comment:
These should be sorted alphabetically by the component name. It helps us to avoid constant PR conflicts when registering new react components.